File: Domain/DataSource/Entities/Ingredient.swift
//
//  Ingredient.swift
//  Domain
//
//  Created by Balázs Kilvády on 2/17/20.
//  Copyright © 2020 kil-dev. All rights reserved.
//

import Foundation

extension DataSource {
    struct Ingredient: Codable {
        typealias ID = Int64

        let id: ID
        let name: String
        let price: Double
    }
}

File: Domain/Sources/CppCart/Ingredient.cpp
//
//  Ingredient.cpp
//  DCPizza
//
//  Created by Balázs Kilvády on 02/09/21.
//  Copyright © 2021. kil-dev. All rights reserved.
//

#include "Ingredient.hpp"

namespace cpplib {

Ingredient::Ingredient(ID id,
                       const string& name,
                       double price)
: id(id), name(name), price(price) {}

}
